docs/usage.md

```diff
@@ -582,7 +547,7 @@ Default: `'none'`.
 > **Tip**: You should provide a small decoy reference genome with pre-made indices, e.g.
 > the human mtDNA genome, for the mandatory parameter `--fasta` in order to
 > avoid long computational time for generating the index files of the reference
-> genome, even if you do not actual need a reference genome for any downstream
+> genome, even if you do not actually need a reference genome for any downstream
 > analyses.
 
 #### `--single_stranded`
```

Comment:
All pipeline parameters should have their documentation in the `nextflow_schema.json` file now, and be removed from `usage.md` completely. This was a big part of the reason for writing the schema - so that all docs use the same source and stuff doesn't get out of sync.